lib/containers/common.pm

```diff
@@ -22,14 +22,15 @@ use warnings;
 use testapi;
 use registration;
 use utils qw(zypper_call systemctl);
+use containers::utils 'can_build_sle_base';
 use version_utils qw(is_sle is_leap is_microos is_opensuse is_jeos is_public_cloud);
 
 our @EXPORT = qw(install_podman_when_needed install_docker_when_needed allow_selected_insecure_registries clean_container_host
-  test_container_runtime test_container_image scc_apply_docker_image_credentials scc_restore_docker_image_credentials);
+  test_container_runtime test_container_image scc_apply_docker_image_credentials scc_restore_docker_image_credentials test_image_zyp_ref_and_verify);
 
 sub install_podman_when_needed {
-    my $host_os = shift;
-    my @pkgs    = qw(podman);
+    my ($host_os, $allow_build) = @_;
```

Comment:
```suggestion
    my ($host_os, $is_sle) = @_;
```

Reply:
done. but i use `is_sle_host` to avoid confusion with the `is_sle` from the version_utils